### ISSUE TITLE
[Bug] After adding device and apply runtime changes the influxdb goes unresponsive

### ISSUE THREAD

**Issue by wurmrobert** [toni-moreno/snmpcollector#403]
- Version: 0.8.0
- OS: Docker container: https://hub.docker.com/r/tonimoreno/snmpcollector/
- Influxdb also with docker: tested with version 1.7.5 and 1.6.6 (same behaviour)

What i did:

- I added a new device, with a measurement group which contains two measurements
- I saved the device
- I updated the device frequency
- Submit and apply changes on runtime

After this procedure the influxdb goes unresponsive. So the collector isn´t able to store any new points and no influxdb queries could be done. I need to restart the influxdb docker container that the db reacts again.

The logs from the influxdb:

```
influxdb_1                         | [httpd] 172.18.0.1 - admin [15/Apr/2019:16:16:02 +0000] "POST /write?consistency=&db=monitoring&precision=s&rp=autogen HTTP/1.1" 500 20 "-" "snmpcollector" c3565c9c-5f99-11e9-808f-0242ac120006 10001992
influxdb_1                         | ts=2019-04-15T16:16:12.339825Z lvl=error msg="[500] - \"timeout\"" log_id=0EpRvqzW000 service=httpd
influxdb_1                         | [httpd] 172.18.0.1 - admin [15/Apr/2019:16:16:22 +0000] "POST /write?consistency=&db=monitoring&precision=s&rp=autogen HTTP/1.1" 500 20 "-" "snmpcollector" cf45a008-5f99-11e9-8091-0242ac120006 10001588
influxdb_1                         | ts=2019-04-15T16:16:32.326224Z lvl=error msg="[500] - \"timeout\"" log_id=0EpRvqzW000 service=httpd
influxdb_1                         | [httpd] 172.18.0.1 - admin [15/Apr/2019:16:16:42 +0000] "POST /write?consistency=&db=monitoring&precision=s&rp=autogen HTTP/1.1" 500 20 "-" "snmpcollector" db2ef461-5f99-11e9-8092-0242ac120006 10001458
influxdb_1                         | ts=2019-04-15T16:16:52.344859Z lvl=error msg="[500] - \"timeout\"" log_id=0EpRvqzW000 service=httpd
influxdb_1                         | [httpd] 172.18.0.1 - admin [15/Apr/2019:16:17:02 +0000] "POST /write?consistency=&db=monitoring&precision=s&rp=autogen HTTP/1.1" 500 20 "-" "snmpcollector" e717d7d3-5f99-11e9-8093-0242ac120006 10001790
influxdb_1                         | ts=2019-04-15T16:17:12.326729Z lvl=error msg="[500] - \"timeout\"" log_id=0EpRvqzW000 service=httpd
influxdb_1                         | [httpd] 172.18.0.1 - admin [15/Apr/2019:16:17:22 +0000] "POST /write?consistency=&db=monitoring&precision=s&rp=autogen HTTP/1.1" 500 20 "-" "snmpcollector" f3070705-5f99-11e9-8094-0242ac120006 10001406
influxdb_1                         | ts=2019-04-15T16:17:32.314394Z lvl=error msg="[500] - \"timeout\"" log_id=0EpRvqzW000 service=httpd
```

Does someone has the same behaviour? Maybe the bug is at the influxdb?

**Comment by wurmrobert** [comment]
Yes i can also reproduce it with version 1.6.6.
After adding a new device and apply the runtime changes the collector adds a new tag at the measurement. So the issue from the influxdb could fit the problem.

The collector only returns that the influx points couldn´t get saved:
```
ERRO[2019-04-17 09:41:52] ERROR on Write batchPoint in DB influx (57 points) | elapsed : 9.994960707s | Error: {"error":"timeout"}
  
ERRO[2019-04-17 09:42:12] ERROR on Write batchPoint in DB influx (57 points) | elapsed : 9.993623075s | Error: {"error":"timeout"}
  
ERRO[2019-04-17 09:42:32] ERROR on Write batchPoint in DB influx (182 points) | elapsed : 9.998479184s | Error: {"error":"timeout"}
```

**Comment by wurmrobert** [comment]
Great news.

InfluxDb has released version 1.7.6 and three hours ago the docker container has been published to docker hub. 

This version has some bugfixes wich no longe have the faulty behaviour. 
So updating to influxdb version 1.7.6 fixed the problem. 
Thanks @sbengo for your help. I think we can close the issue.
